package.json

```diff
@@ -44,25 +45,23 @@
     "url": "https://github.com/proccheck/proccheck-html/issues"
   },
   "homepage": "https://github.com/proccheck/proccheck-html#readme",
-  "devDependencies": {
+  "peerDependencies": {
     "@commitlint/cli": "^7.5.2",
     "@commitlint/config-conventional": "^7.5.0",
-    "@types/jest": "24.0.11",
+    "@types/jest": "^24.0.11",
+    "coveralls": "^3.0.4",
     "cross-env": "^5.2.0",
     "husky": "^1.3.1",
-    "jest": "24.7.1",
+    "lerna": "^3.15.0",
```

Comment:
Si tu enlève ` jest` des dependances mais conserve les commandes qui l'utilisent je pense que ça ne fonctionnera pas à moins que je rate quelque chose

Reply:
maybe this will be updated when we implement the first test to chek if it works well with lerna configuration of the differents packages.
For the moment, It is better to ignore this part until #1 is merged with the code which have a few UT.